src/pages/Shop/Components/ItemDetail/Components/ItemReviewList/Components/ItemReview.js

```diff
@@ -0,0 +1,101 @@
+import React, { Component } from 'react'
+
+import './ItemReview.scss'
+
+class ItemReview extends Component {
+  constructor() {
+    super();
+    this.state = {
+      isHide: true,
+      nestedComment: '',
+      commentList: []
+    }
+  }
+
+  showMore = () => {
+    this.setState({
+      isHide: !this.state.isHide
+    })
+  }
+
+  inputHandler = (e) => {
+    this.setState({
+      nestedComment: e.target.value
+    })
+  }
+
+  addComment = (evt) => {
+    evt.preventDefault();
+    const {commentList, nestedComment} = this.state
+    this.setState({
+      commentList: [...commentList, 
+      {
+        id: commentList.lenght +1,
+        user: 'minddon',
+        comment: nestedComment
+      },
+    ],
+      nestedComment: ''
+    })
+  }
+
+  render() {
+    const {id, option, comment, imgUrl, userId, postedDate} = this.props;
+    const {isHide, commentList, nestedComment} = this.state;
+
+    return(
+      <li className={isHide ? "ItemReview" : "ItemReview opened"}>
```

Comment:
```suggestion
      <li className={`ItemReview ${!isHide && "opened"}`}>
```
false일 때 추가되는 className은 이렇게 할당해주실 수 있습니다.

Reply:
넹 수정하였습니다~